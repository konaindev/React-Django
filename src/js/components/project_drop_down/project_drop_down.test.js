import renderer from "react-test-renderer";
import ProjectDropDown from "./index";

describe("ProjectDropDown", () => {
  const project = { name: "Portland Multi Family", public_id: "pro_example" };

  it("renders without building image correctly", () => {
    const props = {
      project: {
        ...project,
        building_image: null
      }
    };

    const tree = renderer.create(<ProjectDropDown {...props} />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it("renders with building image correctly", () => {
    const props = {
      project: {
        ...project,
        building_image: null
      }
    };

    const tree = renderer.create(<ProjectDropDown {...props} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
