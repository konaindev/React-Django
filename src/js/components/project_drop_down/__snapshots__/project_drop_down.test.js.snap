// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProjectDropDown renders with building image correctly 1`] = `
<div
  className="project-drop-down"
>
  <span
    className="project-drop-down__image"
    style={Object {}}
  />
  <span
    className="project-drop-down__text"
  >
    Portland Multi Family
  </span>
</div>
`;

exports[`ProjectDropDown renders without building image correctly 1`] = `
<div
  className="project-drop-down"
>
  <span
    className="project-drop-down__image"
    style={Object {}}
  />
  <span
    className="project-drop-down__text"
  >
    Portland Multi Family
  </span>
</div>
`;
